src/ros_filter.cpp

```diff
@@ -361,6 +361,25 @@ namespace RobotLocalization
   }
 
   template<typename T>
+  std::string RosFilter<T>::tf2NameSanitizer(const std::string & param )
+  {
+    if(!tfPrefix_.empty())
+    { // tf prefix is set
```

Comment:
Line break here, before the comment (pedantic, I know, but I want to be consistent)